src/cs4450_project/FPCameraController.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * file: FPCameraController.java
+ * author: Isaac Quintanilla
+ * class: CS 4450 - Computer Graphics
+ *
+ * assignment: Project
+ * date last modified: 10/9/2021
+ *
+ * purpose: This class will hold our camera's position in 3D space.
+ */
+package cs4450_project;
+
+import org.lwjgl.util.vector.Vector3f;
+import org.lwjgl.input.Keyboard;
+import org.lwjgl.input.Mouse;
+import org.lwjgl.opengl.Display;
+import static org.lwjgl.opengl.GL11.*;
+import org.lwjgl.Sys;
+
+public class FPCameraController {
+    // include CUBE class for gameloop render
+    Cube cube = new Cube(0, 0, 0, 1);
```

Comment:
```suggestion
    private Cube cube = new Cube(0, 0, 0, 1);
```
I prefer explicit private.